### ISSUE TITLE
basenet solver and train prototxt

### ISSUE THREAD

**Issue by iqbalnaved** [RankingCNN/Using-Ranking-CNN-for-Age-Estimation#5]
Could you please provide the solver and train prototxt for the basenet also ? I am trying to train my own dataset for the basenet. Thanks .

**Comment by RankingCNN** [comment]
Hi @iqbalnaved, you can download the prototxt files for the training of the base CNN from the following links:
https://www.dropbox.com/s/pe55b4kewzbvhwv/solver_base.prototxt?dl=0
https://www.dropbox.com/s/xx6id8e35txsk9m/train_val_base.prototxt?dl=0
Notice that besides different datasets, the other thing different between the base CNN and basic CNNs would be the "num_output" in the last fully-connected layer.

**Comment by iqbalnaved** [comment]
@RankingCNN  So you also have used a validation set for the basenet ? In the paper you only metioned a train set of 26K images, what is the size of the validation set ?

**Comment by RankingCNN** [comment]
@iqbalnaved We tried different ways of using the Adience dataset. You can either use all 26k images for pre-training or use 80% for pre-training and 20% for validation. Either way, it won't make that big a difference after fine-tuning with MORPH dataset.
